chef-config/lib/chef-config/config.rb

```diff
@@ -1221,9 +1221,6 @@ def self.guess_internal_locale
     # can be set to a string or array of strings for URIs to set as rubygems sources
     default :rubygems_url, "https://www.rubygems.org"
 
-    # globally sets the default of the clear_sources property on the gem_package and chef_gem resources
-    default :clear_gem_sources, false
```

Comment:
this should probably be changed to an explicit nil default rather than relying on chef-config autovivification (for documentation purposes if nothing else, but i think we actually consume chef-config and turn on strict checking for something like berkshelf as well -- we always intended to turn on strict checking in chef-client but never did)

Reply:
I wondered about that, I agree it makes sense to be more explicit.